kvwrapper_etcd/kvwrapper_etcd.go

```diff
@@ -24,7 +24,8 @@ func (e EtcdWrapper) NewKVWrapper(servers []string, username, password string) k
 	}
 	client, err := etcd.New(config)
 	if err != nil {
-		panic(err)
+		log.Warn("Could not instatntiate etcd V2 client.", "err", err)
```

Comment:
@PinchasLev consider moving log level from Warn->Crit, as this is essentially a terminal error